app/src/main/java/com/infomaniak/drive/ui/addFiles/ImportFilesDialog.kt

```diff
@@ -73,7 +73,9 @@ class ImportFilesDialog : DialogFragment() {
         super.onDismiss(dialog)
         val errorCount = importCount - successCount
 
-        if (errorCount > 0) {
+        if (errorCount > 0 && isLowMemory()) {
+            showSnackbar(R.string.uploadOutOfMemoryError, showAboveFab = true)
+        } else if (errorCount > 0) {
```

Comment:
Maybe we could write a `if` statement that handles checking if `errorCount > 0` and inside we can check with a second `if` statement which snackbar needs to be displayed